src/common/libschedutil/init.c

```diff
@@ -49,13 +50,50 @@ schedutil_t *schedutil_create (flux_t *h,
     return NULL;
 }
 
+static void respond_to_outstanding_msgs (schedutil_t *util)
+{
+    int rc = 0;
+    flux_future_t *fut;
+    flux_msg_t *msg;
+    for (fut = zlistx_first (util->outstanding_futures);
+         fut;
+         fut = zlistx_next (util->outstanding_futures)) {
+        msg = flux_future_aux_get (fut, "schedutil::msg");
+        rc += flux_respond_error (util->h, msg, ENOSYS,
```

Comment:
Suggest logging an error each time respond fails rather than adding -1's and zeroes and logging once.